modules/serverless-rn-1-22-0.adoc

```diff
@@ -17,7 +17,7 @@
 * {ServerlessProductName} now uses Knative `kn` CLI 1.1.
 * {ServerlessProductName} now uses Knative Kafka 1.1.
 * The `kn func` CLI plug-in now uses `func` 0.23.
-* Knative service specifications now support init containers.
+* The init containers support for Knative services is now available as a Technology Preview.
 * The persistent volume claim (PVC) support for Knative services is now available as a Technology Preview.
```

Comment:
please remove "the" for both lines, e.g.
* Initi containers support...
* Persistent volume claim...
@rh-max I'm OK with merging once this is done